lib/xandra/connection.ex

```diff
@@ -78,6 +81,12 @@ defmodule Xandra.Connection do
                  compressor,
                  options
                ) do
+          :telemetry.execute([:xandra, :connection], %{}, %{
```

Comment:
**Suggestion**: I’m thinking maybe this would sound better as `[:xandra, :connected]` and `[:xandra, :disconnected]`, what do you think? "Connection" sounds more like it's related to the connection itself, not to the event of having connected 🤔